erpnext_shopify/erpnext_shopify/doctype/shopify_settings/shopify_settings.json

```diff
@@ -601,14 +679,15 @@
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
-   "fieldname": "sales_invoice_series", 
-   "fieldtype": "Select", 
+   "description": "If Sync Sales Invoice is enabled then the system will create Sales Invoice against Sales Order whose payment is marked in Shopify.", 
```

Comment:
avoid description. The label should be self explanatory. It should be `Import Delivery Notes from Shopify on Shipment`